apigenerator/resources/1 - Project/1 - BaseProject/Project/src/e_Infra/g_Environment/EnvironmentVariables.py

```
@@ -0,0 +1,75 @@
+# System Imports #
+import os
+
+# ------------------------------------------ Database ------------------------------------------ #
+
+# Database start configuration #
+os.environ['main_db_conn'] = 'pgsql'
+
+# Configuration for database connection #
+os.environ['pgsql_user'] = 'seven-technologies-web-app_owner'
```

Ruff (0.11.9)

Use capitalized environment variable `PGSQL_USER` instead of `pgsql_user`

Replace `pgsql_user` with `PGSQL_USER`

(SIM112)